lib/livebook_cli/server.ex

```diff
@@ -126,6 +127,10 @@ defmodule LivebookCLI.Server do
       |> append_path("/explore/notebooks/new")
       |> Livebook.Utils.browser_open()
     end
+    
+    if Keyword.has_key?(opts, :notebook) do
```

Comment:
There's a neat pattern for optional options:

```elixir
if notebook = opts[:notebook] do
  ...
end
```

Reply:
thanks. incorporating this :)